GVFS/GVFS.Platform.Mac/ProjFSKext.cs

```diff
@@ -18,7 +18,6 @@ public class ProjFSKext : IKernelDriver
 
         public bool IsGVFSUpgradeSupported()
         {
-            // TODO(Mac)

```

Comment:
This is a rather large work item, covering much more than just this one function, so this TODO isn't adding much value. And there is an issue tracking upgrade on Mac.